news/wn_data.go

```
@@ -0,0 +1,320 @@
+package news
+
+var worldNewsData_ID = `
```

golangci-lint (1.62.2)

var `worldNewsData_ID` is unused

(unused)